examples/webRtcTcpDetectEvent/webpack.common.js

```diff
@@ -0,0 +1,69 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const webpack = require('webpack');
+require('dotenv').config({ path: './.env' }); 
+
+module.exports = {
+    entry: {
+        index: './src/index.ts',
+    },
+    plugins: [
+        new HtmlWebpackPlugin({
+            title: 'Scalable Pixel Streaming Frontend',
+            template: './src/index.html',
+            filename: 'index.html'
+        }),
+    ],
+    module: {
+        rules: [
+            {
+                test: /\.tsx?$/,
+                loader: 'ts-loader',
+                exclude: [
+                    /node_modules/,
+                ],
+            },
+            {
+                test: /\.html$/i,
+                use: 'html-loader'
+            },
+            {
+                test: /\.css$/,
+                type: 'asset/resource',
+                generator: {
+                    filename: '[name][ext]'
+                }
+            },
+            {
+                test: /\.(png|svg)$/i,
+                type: 'asset/resource',
+                generator: {
+                    filename: 'images/[name][ext]'
+                }
+            },
+        ],
+    },
+    resolve: {
+        extensions: ['.tsx', '.ts', '.js', '.svg'],
+    },
+    output: {
+        filename: '[name].js',
+        library: 'spstypescriptexample',
```

Comment:
Different output name for the library here perhaps?

Reply:
Changed the library name